File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/BotguardData.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class BotguardData{

	@SerializedName("interpreterUrl")
	private String interpreterUrl;

	@SerializedName("program")
	private String program;

	public void setInterpreterUrl(String interpreterUrl){
		this.interpreterUrl = interpreterUrl;
	}

	public String getInterpreterUrl(){
		return interpreterUrl;
	}

	public void setProgram(String program){
		this.program = program;
	}

	public String getProgram(){
		return program;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/BotguardData.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class BotguardData : Parcelable, Serializable {
    @SerializedName("interpreterUrl")
    var interpreterUrl: String? = null

    @SerializedName("program")
    var program: String? = null
    override fun toString(): String {
        return "BotguardData{" +
                "interpreterUrl = '" + interpreterUrl + '\'' +
                ",program = '" + program + '\'' +
                "}"
    }
}